engine/src/test/java/org/camunda/bpm/engine/test/api/filter/FilterTaskQueryTest.java

```diff
@@ -823,6 +849,63 @@ public void testExtendingTaskQueryWithAssigneeNotIn() {
     assertEquals(0, extendingQueryTasks.size());
   }
 
+  @Test
+  public void testExtendingEmptyTaskQueryWithCandidateGroupLike() {
+    TaskQuery query = taskService.createTaskQuery();
+
+    saveQuery(query);
+
+    List<Task> tasks = filterService.list(filter.getId());
+    // 3 test tasks have been created during setup (no filters -> we expect all tasks to be found)
```

Comment:
Could you please adapt the comments of this test (and also the two tests below) to follow the `// given // when // then` pattern that we use in other tests?

Reply:
fixed